yt_dlp/extractor/eporner.py

```diff
@@ -1,3 +1,4 @@
+import re
 from .common import InfoExtractor
```

Comment:
won't be needing it if my other suggestion is applied
```suggestion
from .common import InfoExtractor
```